postgres.go

```diff
@@ -106,11 +106,12 @@ func (s *PostgresMutex) lock() error {
 			time.Sleep(jitter)
 		}
 
-		if s.tx, err = s.client.Begin(); err != nil {
+		s.tx, err = s.client.Begin(context.TODO())
```

Comment:
if we are not planning to add contexts later, just use `context.Background()`

Reply:
Done